components/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react'
+import { IoIosArrowRoundForward } from "react-icons/io";
+import Button from 'components/Button';
+import Search from "assets/icons/svg/search-icon.svg"
+
+const NotFound: FC = () => {
+  return (
+    <div className="pt-6 md:p-0 text-xl w-full my-auto h-[calc(100vh-450px)] md:h-[calc(100vh-270px)]">
```

Comment:
A few adjustments are required for the mobile view.

<img width="361" alt="Screenshot 2023-12-30 at 8 14 28 PM" src="https://github.com/rupali-codes/LinksHub/assets/93012310/f3231f82-ddcf-41a0-a48a-c24785b43d32">

Maybe you can utilize `min-height` to fix this. 

```suggestion
    <div className="pt-6 md:p-0 text-xl w-full my-auto min-h-[calc(100vh-450px)] md:min-h-[calc(100vh-270px)]">
```